board/bootmode.h

```diff
@@ -20,7 +20,7 @@ void jump_to_bootloader(void) {
   NVIC_SystemReset();
 }
 
-void early(void) {
+void bootmode(void) {
```

Comment:
I agree early wasn't a great name, but 'bootmode' also doesn't say much imo

Reply:
Mmm, ideas? Because as I understood the purpose of that function is to check boot flag and select how to boot - normal mode, flasher or DFU. I don't see something special there about hardware that must be initialized before main.c

Reply:
what do you mean? it does a lot of stuff:
- disable interrupts
- initialize the register driver
- decide the boot more
- check that it's running on the right chip
- setup gpio and the interrupt table
- check which board we're running on

let's name it 'early_initialization' or something, no?

Reply:
So why do we initialize peripherals, clocks and etc again in main.c?

Reply:
Ok, will name it as 'early_initialization'.